src/jwtf.erl

```diff
@@ -269,14 +269,20 @@ split(EncodedToken) ->
     end.
 
 
-decode_json(Encoded) ->
-    case b64url:decode(Encoded) of
-        {error, Reason} ->
-            throw({bad_request, Reason});
-        Decoded ->
-            jiffy:decode(Decoded)
+decode_b64_json(B64Encoded) ->
+    try
+        case b64url:decode(B64Encoded) of
+            {error, Reason} ->
+                throw({bad_request, Reason});
+            JsonEncoded ->
```

Comment:
b64url:decode is quite naughty for having a `binary() | {error, term()}` result, but it is what it is...

Reply:
yeah, it's super annoying that it can both return an `{error, term()}` tuple and also throw exceptions